iroha/src/sumeragi.rs

```diff
@@ -899,19 +1104,20 @@ mod tests {
             task::spawn(async move {
                 torii.start().await.expect("Torii failed.");
             });
+            let config = Configuration {
+                private_key: keys[i].1,
+                peers: &ids,
+                peer_id: ids[i].clone(),
+                max_faults,
+                blocks_sender: Arc::new(RwLock::new(block_sender)),
+                world_state_view: wsv,
+                transactions_sender,
+                commit_time_ms: config::DEFAULT_COMMIT_TIME_MS,
+                tx_receipt_time_ms: config::DEFAULT_TX_RECEIPT_TIME_MS,
+                block_time_ms: config::DEFAULT_BLOCK_TIME_MS,
+            };
```

Comment:
just use some constructor and hide inner constants

Reply:
You mean in general to remove configuration structure?

Reply:
It was introduced in the first place as the method `Sumeragi::new` had too many arguments. As suggested by clippy and I also agree with it.

Reply:
The constants are actually only used in tests